### PR TITLE
Created and tested NewUserMailer to send an email when a user signs up.

### PR DESCRIPTION
**Pull request by Rdore88**

### CODE REVIEW

File: config/environments/test.rb [hunk 1]
```diff
@@ -34,6 +34,7 @@
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
 
+
```
Comment (rposborne) [inline config/environments/test.rb]: Super critical whitespace add.

File: app/controllers/api/users_controller.rb [hunk 2]
```diff
@@ -6,6 +6,7 @@ def create
     @new_user.secure_random
     if @new_user.save
       render json: {authorization: @new_user.authorization_token}
+      NewUserMailer.sign_up_email(@new_user).deliver_now
```
Comment (rposborne) [inline app/controllers/api/users_controller.rb]: Do we really want this to block?   Could we break this work out into a deliver_later

**Review by rposborne — COMMENTED** [review]
See comments.

File: app/mailers/application_mailer.rb [hunk 3]
```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: "careeratlas1@gmail.com"
```
Comment (rposborne) [inline app/mailers/application_mailer.rb]: Do we have this as an ENV var? Might be better to only have to change the email in one place.

**Review by rposborne — COMMENTED** [review]
One final nitpick.

**Review by rposborne — APPROVED** [review]
🚢

**Comment by jenniferoakes** [comment]
🚢 per Russell

**Pull request merged**